project_resstock_multifamily/housing_characteristics/Geometry Foundation Type.tsv

```diff
@@ -28,13 +28,13 @@ CO_Pueblo.Mem.AP.724640	Single-Family Detached	0	0.19	0.61	0.19	0.01
 CT_Bridgeport-Sikorsky.Mem.AP.725040	Single-Family Detached	0	0.6	0.36	0.02	0.02
 CT_Hartford-Bradley.Intl.AP.725080	Single-Family Detached	0	0.6	0.36	0.02	0.02
 DE_Wilmington-New.Castle.County.AP.724089	Single-Family Detached	0	0.12	0.19	0.41	0.29
-FL_Daytona.Beach.Intl.AP.722056	Single-Family Detached	0	0	0	0.05	0.95
-FL_Jacksonville.Intl.AP.722060	Single-Family Detached	0	0	0	0.05	0.95
-FL_Key.West.Intl.AP.722010	Single-Family Detached	0	0	0	0.05	0.95
-FL_Miami.Intl.AP.722020	Single-Family Detached	0	0	0	0.05	0.95
-FL_Orlando.Intl.AP.722050	Single-Family Detached	0	0	0	0.05	0.95
-FL_Tallahassee.Rgnl.AP.722140	Single-Family Detached	0	0	0	0.05	0.95
-FL_West.Palm.Beach.Intl.AP.722030	Single-Family Detached	0	0	0	0.05	0.95
+FL_Daytona.Beach.Intl.AP.722056	Single-Family Detached	0	0.0	0.0	0.05	0.95
```

Comment:
I don't understand why the decimal place is added to some cells but not others. I guess "0" should be preferred over "0.0" since it takes up fewer bytes.